### PR TITLE
chore(root): fixing `root` semantics in convert_dataset script

### PR DESCRIPTION
**Pull request by CarolinePascal**
## Type / Scope

- **Type**: Chore
- **Scope**: `convert_dataset_v21_to_v30` script

## Summary / Motivation

Following up on #3035, this PR fixes the interpretation of the `root` argument in the `convert_dataset_v21_to_v30` script.

## Related issues

- Fixes / Closes: # (if any)
- Related: # (if any)

## What changed

- `root` now defines the exact dataset folder containing meta/, data/, videos/.

## How was this tested (or how to run locally)

Successfully converted a local dataset using the `--root` argument.

## Checklist (required before merge)

- [ ] Linting/formatting run (`pre-commit run -a`)
- [ ] All tests pass locally (`pytest`)
- [ ] Documentation updated
- [ ] CI is green

## Reviewer notes

- Anything the reviewer should focus on (performance, edge-cases, specific files) or general notes.
- Anyone in the community is free to review the PR.

### CODE REVIEW

File: src/lerobot/datasets/v30/convert_dataset_v21_to_v30.py [hunk 1]
```diff
@@ -38,6 +38,9 @@
     --repo-id=lerobot/pusht \
     --root=/path/to/local/dataset/directory
```
Comment (s1lent4gnt) [inline src/lerobot/datasets/v30/convert_dataset_v21_to_v30.py]: Nice to add backslash here at the end.

**Review by s1lent4gnt — APPROVED** [review]
Thank you for checking this, LGTM!

File: src/lerobot/datasets/v30/convert_dataset_v21_to_v30.py (hunk 1 above)
Comment (CarolinePascal) [reply src/lerobot/datasets/v30/convert_dataset_v21_to_v30.py]: Done in [2d403ba](https://github.com/huggingface/lerobot/pull/3073/commits/2d403bae112862649653abda6363fc4e9337cdda)
